app/routes/datasets/downloads.py

```diff
@@ -174,8 +175,14 @@ async def download_csv_post(
 
     await _check_downloadability(dataset, version)
 
+    geostore: Optional[GeostoreCommon] = None
+    if request.geometry:
+        geostore = GeostoreCommon(
+            geojson=request.geometry, geostore_id=uuid4(), area__ha=0, bbox=[0, 0, 0, 0]
```

Comment:
Should area__ha and bbox be optional given this use case when you are forced to pass fake values?

Reply:
Hmm, good question. It does feel like an abuse of the class, since everything but the geometry is fake. I'll create a ticket to look into improving it, thanks!

Reply:
https://gfw.atlassian.net/browse/GTC-2556